sdk/storage/ustorageclient/build_all/configs/ustorageclient_functions.cmake

```diff
@@ -0,0 +1,189 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#Macro to set C and Cpp standards for a given target
+macro(add_standards)
+    set(CMAKE_C_STANDARD                99)
+    set(CMAKE_C_STANDARD_REQUIRED       OFF)
+    set(CMAKE_C_EXTENSIONS              OFF)
+
+    set(CMAKE_CXX_STANDARD              11)
```

Comment:
I would prefer just setting this in the CMakeLists file instead of using a macro